Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/InteractableToggleCollection.cs

```diff
@@ -11,78 +12,108 @@ namespace Microsoft.MixedReality.Toolkit.UI
     /// </summary>
     public class InteractableToggleCollection : MonoBehaviour
     {
-        [Tooltip("Interactables that will be managed by this controller")]
+        [Tooltip("Array of Interactables that will be managed by this controller")]
+        /// <summary>
+        /// Array of Interactables that will be managed by this controller
+        /// </summary>
         public Interactable[] ToggleList;
```

Comment:
ToggleList does not have get/set and backing serialized private field etc

Reply:
Added to ToggleList:
```
        [Tooltip("Array of Interactables that will be managed by this controller")]
        [SerializeField, FormerlySerializedAs("ToggleList")]
        private Interactable[] toggleList;

        /// <summary>
        /// Array of Interactables that will be managed by this controller
        /// </summary>
        public Interactable[] ToggleList
        {
            get => toggleList;
            set => toggleList = value;
        }
```

Reply:
You need to update the selection on the set. If I change the size of the list, then my current index may be invalid

Reply:
Addressed in PR #6523